velero/spectrum-fusion/cert-manager/backup-restore-workflow.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: application.isf.ibm.com/v1alpha1
+kind: Application
+metadata:
+  name: cert-manager-application
+  namespace: ibm-spectrum-fusion-ns
+spec:
+  enableDR: false
+  includedNamespaces:
+    - <cert manager namespace>
+    - openshift-marketplace    
+    - openshift-config
+---
+
+apiVersion: data-protection.isf.ibm.com/v1alpha1
+kind: BackupPolicy
+metadata:
+  name: cert-manager-backup-policy
+  namespace: ibm-spectrum-fusion-ns
+spec:
+  backupStorageLocation: <storage_location>
+  provider: isf-backup-restore
+  retention:
+    number: 5
+    unit: days
+  schedule:
+    cron: '00 0  * * * '
+    timezone: America/New_York
+---
+
+apiVersion: spp-data-protection.isf.ibm.com/v1alpha1
+kind: Recipe
+metadata:
+  name: cert-manager-recipe
+  namespace: ibm-spectrum-fusion-ns
+spec:
+  appType: cert-manager
+  groups:
+    - includeClusterResources: true
+      includedResourceTypes:
+        - certmanagerconfigs.operator.ibm.com
+        - customresourcedefinitions.apiextensions.k8s.io
+      labelSelector: foundationservices.cloudpak.ibm.com=cert-manager
+      name: cert-manager-resources
+      type: resource
+    - backupRef: cert-manager-resources
+      includeClusterResources: true
+      includedResourceTypes:
+        - customresourcedefinitions.apiextensions.k8s.io
+      name: cert-manager-config-crd
+      type: resource
+    - backupRef: cert-manager-resources
+      includeClusterResources: true
+      includedResourceTypes:
+        - certmanagerconfigs.operator.ibm.com
+      name: cert-manager-config-cr
+      type: resource
+    - includedResourceTypes:
+        - catalogsources.operators.coreos.com
+      labelSelector: foundationservices.cloudpak.ibm.com=catalog
+      name: common-services-catalogs
+      type: resource
+    - includedNamespaces:
+        - openshift-config
+      includedResourceTypes:
+        - secrets
+      labelSelector: foundationservices.cloudpak.ibm.com=pull-secret
+      name: pull-secret
+      type: resource
+    - backupRef: pull-secret
+      includedNamespaces:
+        - openshift-config
+      includedResourceTypes:
+        - secrets
+      labelSelector: foundationservices.cloudpak.ibm.com=pull-secret
+      name: ow-pull-secret
+      restoreOverwriteResources: true
+      type: resource
+    - includeClusterResources: true 
+      labelSelector: foundationservices.cloudpak.ibm.com=namespace
+      name: common-services-namespace
```

Comment:
Do we want to include CS namespaces in this recipe? We would have to increase the scope of the Application CR to include them and it would make this recipe less-independent from the regular CS recipe. Same goes for the operatorgroup below. I guess we are still restoring the cert manager namespace and its operatorgroup and both those resources share a label with the CS namespaces and operatorgroups but I think we can at least call these steps cert-manager-namespace and cert-manager-operatorgroup. The namespace scope defined in the Application CR should exclude any of these resources in namespaces not included